core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -1794,7 +1794,7 @@ class SubmissionMonitorSpec(_system: ActorSystem)
         })).flatten
         submissionList.forall(_.status == SubmissionStatuses.Done.toString) && submissionList.length == numSubmissions
       },
-      max = 30 seconds,
+      max = 60 seconds,
```

Comment:
This is a known sporadic test failure-- we may not want to increase the timeout interval, but hopefully the JIRA ticket will provide more information (https://broadinstitute.slack.com/archives/C03BCARP494/p1669048189832469).

Reply:
Workflows team has OK'd this timeout increase. https://broadinstitute.slack.com/archives/C1EH66VCM/p1669216629798999